verilog/formatting/align.cc

```diff
@@ -66,6 +66,15 @@ static bool TokensAreAllComments(const T& tokens) {
              }) == tokens.end();
 }
 
+template <class T>
+static bool TokensHaveParenthesis(const T& tokens) {
+  return std::find_if(
+              tokens.begin(), tokens.end(),
+              [](const typename T::value_type& token) {
+                return token.TokenEnum() == '(';
+              }) == tokens.end();
```

Comment:
The name of this function is HaveParen, but the logic here (iterator == end()) is the inverse "not-have".

Reply:
also c++11 introduces `std::any_of` so you don't have to compare against `end()`